### DISCUSSION TITLE
fullScreenModal is sometimes full screen and sometimes has a gap at the top.

### DISCUSSION THREAD

**Discussion by timurridjanovic** [software-mansion/react-native-screens#815] [Q&A]
On ios the fullScreenModal stackPresentation generally doesnt take up the entire screen and you can dismiss it by just swipping down the top of the modal. That said, on an iPhone 5, it does take the entire screen, so I'd need to add a close button there. Is there a way to detect whether the modal will take up the entire screen or not?

**Comment by WoLewicki** [comment]
It should always take the entire screen. Can you post it as an issue since it rather a bug?

**Comment by tboba** [comment]
Hi @timurridjanovic, I'm closing this thread because of inactivity, but if you're still facing this problem don't hesitate to create an issue with steps to reproduce (including minimal repro).

Cheers!
